channels/api/http/endpoint_test.go

```diff
@@ -140,6 +152,18 @@ func TestCreateChannelEndpoint(t *testing.T) {
 			status:      http.StatusBadRequest,
 			err:         apiutil.ErrNameSize,
 		},
+		{
+			desc:     "create channel with invalid route format",
+			token:    validToken,
+			domainID: validID,
+			req: channels.Channel{
+				Name:  valid,
+				Route: "__invalid",
```

Comment:
Also, add a check for UUID format. We must not allow UUIDs in routes to prevent confusion and make our lives easier.

Reply:
Add test here as well:
```go
		{
			desc:     "create channel with UUID route",
			token:    validToken,
			domainID: validID,
			req: channels.Channel{
				Name:  valid,
				Route: testsutil.GenerateUUID(t),
			},
			contentType: contentType,
			status:      http.StatusBadRequest,
			err:         apiutil.ErrInvalidRouteFormat,
		},
```
Validation testing is fine, but it only works if **decoding request works properly**, so we to test it here as well.